libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -623,7 +626,9 @@ std::vector<btc_txout> bitcoin_rpc_client::listunspent_by_address_and_amount(con
             btc_txout txo;
             txo.txid_ = entry.second.get_child("txid").get_value<std::string>();
             txo.out_num_ = entry.second.get_child("vout").get_value<unsigned int>();
-            txo.amount_ = entry.second.get_child("amount").get_value<double>();
+            string amount = entry.second.get_child("amount").get_value<std::string>();
+            amount.erase(std::remove(amount.begin(), amount.end(), '.'), amount.end());
+            txo.amount_ = std::stoll(amount);
             result.push_back(txo);
```

Comment:
why not use libbitcoin function btc_to_satoshi?

Reply:
Now libbitcoin is used only from *_standalone functions,  this is rpc call that will be used even if we will not use libbitcoin in the future.